tensorboard/components/polymer_config.ts

```diff
@@ -13,6 +11,5 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
--->
-
-<script src="registry.js"></script>
+==============================================================================*/
+(window as any).polymerSkipLoadingFontRoboto = true;
```

Comment:
Who is loading Roboto now?

Reply:
This is loading one from CDN and it is a behavior change we did not intend. 

We should load the Roboto from our own backend instead.